spyder/plugins/preferences/plugin.py

```diff
@@ -25,8 +25,6 @@
 from spyder.config.base import _
 from spyder.config.main import CONF_VERSION
 from spyder.config.user import NoDefault
-from spyder.plugins.preferences.api import PreferencePages, GeneralConfigPage
```

Comment:
If `GeneralConfigPage` is not necessary anymore, please remove it to not leave dead code around.

Reply:
It is being used by the LSP